packages/core/src/types/arazzo.ts

```diff
@@ -87,7 +87,7 @@ const parameter = {
     {
       type: 'object',
       properties: {
-        in: { type: 'string', enum: ['header', 'query', 'path', 'cookie'] },
+        in: { type: 'string', enum: ['header', 'query', 'path', 'cookie', 'body'] },
```

Comment:
Can we add a Spot-specific rule that will error on this?

Reply:
In future -yes.
There should be a list of similar rules